usaspending_api/api_contracts/contracts/award/idv/Activity.md

```diff
@@ -0,0 +1,102 @@
+                        "obligated_amount": 8000.0,
+                        "awarded_amount": 20000.0,
+                        "period_of_performance_start_date": "2016-01-14",
+                        "piid": "INR17PA00008",
+                        "recipient_name": "Booz Allen Hamilton",
+                        "recipient_id": "543ee6af-9096-f32a-abaa-834106bead6a-P",
+                        "grandchild": false
+                    },
+                    {
+                        "award_id": 69054107,
+                        "awarding_agency": "GENERAL SERVICES ADMINISTRATION (GSA)",
+                        "awarding_agency_id": 634,
+                        "generated_unique_award_id": "CONT_AW_4732_4730_GS33FCA001_GS23F0170L",
+                        "last_date_to_order": "2017-09-30",
+                        "obligated_amount": 2257.24,
+                        "awarded_amount": 10000.0,
+                        "period_of_performance_start_date": "2014-10-01",
+                        "piid": "GS33FCA001",
+                        "recipient_name": "Booz Allen Hamilton",
+                        "recipient_id": "9a277fc5-50fc-685f-0f77-be0d96420a17-C",
+                        "grandchild": false
+                    },
+                    {
+                        "award_id": 69216438,
+                        "awarding_agency": "DEPARTMENT OF AGRICULTURE (USDA)",
+                        "awarding_agency_id": 153,
+                        "generated_unique_award_id": "CONT_AW_12D2_4730_AG3151B140009_GS23F0170L",
+                        "last_date_to_order": "2015-04-06",
+                        "obligated_amount": 47840.0,
+                        "awarded_amount": 12000.0,
+                        "period_of_performance_start_date": "2014-04-07",
+                        "piid": "AG3151B140009",
+                        "recipient_name": "Booz Allen Hamilton",
+                        "recipient_id": "9a277fc5-50fc-685f-0f77-be0d96420a17-C",
+                        "grandchild": true
+                    }
+                ],
+                "page_metadata": {
+                    "count": 40,
+                    "page": 1,
+                }
+            }
+
+# Data Structures
+
+## PageMetaData (object)
++ page: 2 (required, number)
++ count: 40 (required, number)
+    The total number of results.
+
+## ChildAward (object)
++ `award_id`: `69054107` (required, string)
+    Unique internal surrogate identifier for an award.  Deprecated.  Use `generated_unique_award_id`.
++ `generated_unique_award_id`: `CONT_AW_1540_NONE_DJB30605051_NONE` (required, string)
+    Unique internal natural identifier for an award.
++ `awarding_agency`: `GENERAL SERVICES ADMINISTRATION (GSA)` (required, string)
++ `awarding_agency_id`: 634 (required, number)
++ `last_date_to_order`: `2017-09-30` (required, string, nullable)
++ `obligated_amount`: 2257.24 (required, number)
++ `awarded_amount`: 10000.00 (required, number)
```

Comment:
I'm open to changing the names for `obligated_amount` and `awarded_amount` to more closely match the database, this is what's used in the mockup.

Reply:
When we were working on this we avoided specific values altogether in order to separate contracts and integration tests. We may want to have another chat about this between teams if there's contention.